package-lock.json

```diff
@@ -366,12 +366,35 @@
         "unist-util-find-all-after": "^3.0.2"
       }
     },
+    "@types/bluebird": {
```

Comment:
It strange, that `package-lock.json` has been changes, because there are no any new packages. Check it